exasol_script_languages_container_tool/lib/api/get_language_activation_builder.py

```diff
@@ -96,14 +62,43 @@ def get_language_activation_builder(
 
     if custom_aliases is None:
         custom_aliases = dict()
+    with open(Path(flavor_path) / "flavor_base" / "language_definition") as f:
```

Comment:
So you decommissioned the LanguageContainerDefinition

Reply:
Yes, it's more difficult to try to re-use, than to do it's own implmenetation